Compiled from "Exception.scala"
public final class scala.util.control.Exception$$anonfun$catching$1 extends scala.runtime.AbstractFunction1<java.lang.Class<?>, java.lang.String> implements scala.Serializable {
  public static final long serialVersionUID;

  public final java.lang.String apply(java.lang.Class<?>);
    Code:
       0: aload_1
       1: invokevirtual #25                 // Method java/lang/Class.getName:()Ljava/lang/String;
       4: areturn

  public final java.lang.Object apply(java.lang.Object);
    Code:
       0: aload_0
       1: aload_1
       2: checkcast     #21                 // class java/lang/Class
       5: invokevirtual #32                 // Method apply:(Ljava/lang/Class;)Ljava/lang/String;
       8: areturn

  public scala.util.control.Exception$$anonfun$catching$1();
    Code:
       0: aload_0
       1: invokespecial #38                 // Method scala/runtime/AbstractFunction1."<init>":()V
       4: return
}
